### PR TITLE
Creating navigation system

### PR DESCRIPTION
**Pull request by dev-AshishRanjan**
Features added in this PR:
1. Created state based routing for Member page and Event Page.
2. Created the basic UI for CMS and navigation panel.
3. Initialized the Member page's  rendering file.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+      document.querySelector("#eve").classList.remove("activeLink");
+    }
+  }, [pageRoute]);
   return (
-    <div>
-<FetchFromApi_event/>
-<FetchFromApi_member/>
+    <div className="App">
+      <div className="controlBoard">
+        <p id="eve" onClick={() => setPageRoute(!pageRoute)}>
```
Comment (inrittik) [inline src/App.js]: Use conditional classes here....avoid manipulation of Real DOM

File: src/components/memberGrid.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <div className="table">
+      {data.length > 0 && data.map((member) => <RowElement data={member} />)}
+    </div>
+  );
+};
+
+const RowElement = ({ data }) => {
+  return <div className="row">
+    <img src={data.} alt="img" />
```
Comment (inrittik) [inline src/components/memberGrid.js]: Do not leave src value undefined....for now, use a placeholder image

Comment (dev-AshishRanjan) [reply src/components/memberGrid.js]: This has been solved. It  was left due to lack of backend api.

File: src/App.js (hunk 1 above)
Comment (dev-AshishRanjan) [reply src/App.js]: In the updated version conditional classes are used.

**Review by inrittik — APPROVED** [review]

**Review by inrittik — APPROVED** [review]

**Pull request merged**